pom.xml

```diff
@@ -4987,7 +4987,7 @@
       <dependency>
         <!-- unused dependency, declared to exclude affinity -->
         <groupId>net.openhft</groupId>
-        <artifactId>chronicle-threads</artifactId>
+        <artifactId>chronicle-Threads</artifactId>
```

Comment:
@bdelbosc Is there a specific reason why this would be uppercase please?
It's been making my eclipse java build fail for not obvious reason...